operations/sat/sat_in_csm.md

```diff
@@ -0,0 +1,59 @@
+
+- The `sat-install-utility` container image is only available with the full SAT product stream. This container image
+  provides uninstall and activate functionality when used with the `prodmgr` command. (In SAT 2.3 and older, SAT was
+  only available to install as a separate product stream. Because these versions were packaged with
+  `sat-install-utility`, it is still possible to uninstall these versions of SAT.)
+
+- The `docs-sat` RPM package is only available with the full SAT product stream. You can find SAT documentation at
+  the links below (see [SAT documentation](#sat-documentation)).
+
+- The `sat-config-management` git repository in Gitea (VCS) and thus the SAT layer of NCN CFS configuration is
+  only available with the full SAT product stream.
+
+If the SAT product stream is not installed, there will be no configuration content for SAT in VCS. Therefore, CFS
+configurations that apply to NCNs (for example, `ncn-personalization`) should not include a SAT layer.
+
+The SAT configuration layer modifies the permissions of files left over from prior installations of SAT, so that the
+Keycloak username that authenticates to the API gateway cannot be read by users other than `root`. Specifically, it
+it does the following:
+
+- Modifies the `sat.toml` configuration file which contains the username so that it is only readable by `root`.
+
+- Modifies the `/root/.config/sat/tokens` directory so that the directory is only readable by `root`. This is needed
+  because the names of the files within the `tokens` directory contain the username.
+
+Regardless of the SAT configuration being applied, passwords and the contents of the tokens are never readable by other
+users. These permission changes only apply to files created by previous installations of SAT. In the current version of
+SAT all files and directories are created with the appropriate permissions.
+
+## SAT documentation
+
+For full SAT documentation, see the [SAT Documentation](https://cray-hpe.github.io/docs-sat/).
+
+If SAT has not been installed before, some initial configuration is required, for example, to authenticate to the API
+gateway with `sat auth`. To complete the initial configuration of SAT, complete the following processes:
+
+- [SAT Authentication](https://cray-hpe.github.io/docs-sat/en-24/install/#sat-authentication)
+- [Generate SAT S3 Credentials](https://cray-hpe.github.io/docs-sat/en-24/install/#generate-sat-s3-credentials)
+- [Set System Revision Information](https://cray-hpe.github.io/docs-sat/en-24/install/#set-system-revision-information)
+
+If the full SAT product stream is not being installed, then it is recommended that you uninstall old versions of the
+SAT product stream to avoid confusion in the output of `sat showrev`.
+
+See [Uninstall: Removing a Version of SAT](https://cray-hpe.github.io/docs-sat/en-24/install/#uninstall-removing-a-version-of-sat)
+for more information.
```

Comment:
```suggestion
SAT product stream to avoid confusion in the output of `sat showrev`. For more information, refer to **Uninstall: Removing a Version of SAT** in the SAT documentation.
```